website/pages/en/querying/graphql-api.mdx

```diff
@@ -168,6 +168,56 @@ This can be useful if you are looking to fetch only entities whose child-level e
 }
 ```
 
+#### Logical operators
+
+You can group multiple parameters in the same `where` argument using the `and` or the `or` operators to filter results based on more than one criteria.
+
+##### `AND` Operator
+
+In the following example, we are filtering for challenges with `outcome` `succeeded` and `number` greater than or equal to `100`.
+
+```graphql
+{
+  challenges(where: { and: [{ number_gte: 100 }, { outcome: "succeeded" }] }) {
+    challenger
+    outcome
+    application {
+      id
+    }
+  }
+}
+```
+
+> **Syntactic sugar:** You can simplify the above query by removing the `and` operator by passing a sub-expression separated by commas.
+>
+> ```graphql
+> {
+>   challenges(where: { number_gte: 100, outcome: "succeeded" }) {
+>     challenger
+>     outcome
+>     application {
+>       id
+>     }
+>   }
+> }
+> ```
+
+##### `OR` Operator
+
+In the following example, we are filtering for challenges with `outcome` `succeeded` or `number` greater than or equal to `100`.
+
+```graphql
+{
+  challenges(where: { or: [{ number_gte: 100 }, { outcome: "succeeded" }] }) {
+    challenger
+    outcome
+    application {
+      id
+    }
+  }
+}
+```
+
```

Comment:
I think worth clarifying that application of many "Or" operators may result in slow queries cc @lutter